### DISCUSSION TITLE
（求老社区帖）关于chroot容器 & 想法征集

### DISCUSSION THREAD

**Discussion by SkySight-666** [PenUniverse/PenMods#261] [奇技淫巧]
如果我没记错的话，老社区应该有提到chroot容器

自然想整点活

比如说这个 [浏览器内核](https://github.com/44670/FBrowser)

但是呢，yd挥了挥大斧头，给S6内存砍成了这个鸟样
![image](https://github.com/user-attachments/assets/eaa23f9c-ebd5-46a2-b76f-206d8f81965e)

所以也不到能干啥了

大家有什么想法没

（PS 词典笔OS的“应用” Miniapp 是以二进制文件包的形式存在的，由Javascript源文件（可能是QuickJS）编译而来，没有逆向思路，自然也没有开发新App的思路 ）

**Comment by Redbeanw44602** [comment]
有逆向思路啊

首先那个二进制是QuickJS Bytecode，目前社区还没有QuickJS字节码的反编译器，有点蛋疼，但可以自行读码。

第二你写Mod其实不明白原厂JS里有什么也不要紧，反编译runner挖接口即可（）好像叫什么libfalcon，时间太久我也忘了。

总之我希望社区多提包，，发到群的Topic里，这样就算没人给我买有词典笔OS的笔我有时间了也能看看研究......

**Reply by SkySight-666** [reply]
> 有逆向思路啊
> 
> 首先那个二进制是QuickJS Bytecode，目前社区还没有QuickJS字节码的反编译器，有点蛋疼，但可以自行读码。
> 
> 第二你写Mod其实不明白原厂JS里有什么也不要紧，反编译runner挖接口即可（）好像叫什么libfalcon，时间太久我也忘了。
> 
> 总之我希望社区多提包，，发到群的Topic里，这样就算没人给我买有词典笔OS的笔我有时间了也能看看研究......

我手上有一个P6的包 我贴这里
123盘开始限流了
关于tg...我没有tg账号 收不到验证码，蛋疼。

```
   "deltaUrl": "http://iotdown.mayitek.com/1687695728/9502418/0c05bb6e-e8ea-4970-ac9f-57a2f163de1e.img",
            "deltaID": "9502418",
            "fileSize": 2125597258,
            "md5sum": "380890a162fbe81603ff42ea2ae55548",
```

**Reply by wyxdlz54188** [reply]
不知道为什么全量只有500多mb
大佬如果可以的话帮忙改一下rootfs的密码
我这试了很多方法都无法写入，或者就是不能打包成img

**Reply by Redbeanw44602** [reply]
可以麻烦将原始的检查更新请求发到我邮箱吗？（不要公开发到这里）

**Reply by Redbeanw44602** [reply]
另外收不到验证码可以尝试用 TelegramX

**Reply by SkySight-666** [reply]
> 可以麻烦将原始的检查更新请求发到我邮箱吗？（不要公开发到这里）
可以

**Comment by wyxdlz54188** [comment]
被管理踢了doge，不是我不思考网上文献翻烂了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月3日(周日) 晚上7:32
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PenUniverse/PenMods-release] （求老社区帖）关于chroot容器 &amp; 想法征集 (Discussion #261)




 
另外收不到验证码可以尝试用 TelegramX

**Comment by wyxdlz54188** [comment]
你的邮箱在哪里看



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月3日(周日) 晚上7:32
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PenUniverse/PenMods-release] （求老社区帖）关于chroot容器 &amp; 想法征集 (Discussion #261)




 
另外收不到验证码可以尝试用 TelegramX

**Reply by Redbeanw44602** [reply]
GitHub 个人主页就有

**Reply by wyxdlz54188** [reply]
你好，有进展了吗这个A6pro是否可以添加penmod

**Reply by SkySight-666** [reply]
> 你好，有进展了吗这个A6pro是否可以添加penmod

不能
当然你可以为 @Redbeanw44602 提供设备让他考虑适配

**Comment by DSFdsfWxp** [comment]
#267 应求

**Reply by SkySight-666** [reply]
谢谢大佬
有了它就可以玩很多好玩的东西了
